<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSetting</class>
 <widget class="QWidget" name="FormSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="mainFrame">
     <property name="styleSheet">
      <string notr="true">#mainFrame {
border: 1px solid black;
border-radius: 1px;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>بازنشانی تنظیمات :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnResetToDefault">
          <property name="toolTip">
           <string>بازگرداندن تنظیمات به حالت اولیه</string>
          </property>
          <property name="icon">
           <iconset resource="rc/rc.qrc">
            <normaloff>:/icons/reset_to_default.png</normaloff>:/icons/reset_to_default.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line3_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>کلید رمزنگاری</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnChangeKey">
            <property name="icon">
             <iconset resource="rc/rc.qrc">
              <normaloff>:/icons/change_key.png</normaloff>:/icons/change_key.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSpinBox" name="spbKey">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <number>10000001</number>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkShowAtSystemtray">
        <property name="text">
         <string>نمایش در کنار ساعت سیستم هنگام بسته شدن</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkRunAtWindowsStartup">
        <property name="text">
         <string>اجرای برنامه هنگام شروع به کار ویندوز</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>حاشیه کیوآر کد :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spbBorderSize">
          <property name="toolTip">
           <string>اندازه فاصله کیوآر کد با کادر حاشیه</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>ابعاد کیوآر کد :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spbQRCodeSize">
          <property name="toolTip">
           <string>اندازه کیوآر کد در زمان ذخیره به عنوان عکس</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <number>150</number>
          </property>
          <property name="maximum">
           <number>900</number>
          </property>
          <property name="value">
           <number>150</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>رنگ کیوآر کد :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnQRCodeColorChoice">
          <property name="toolTip">
           <string>انتخاب رنگ</string>
          </property>
          <property name="icon">
           <iconset resource="rc/rc.qrc">
            <normaloff>:/icons/color-choice.png</normaloff>:/icons/color-choice.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rc/rc.qrc"/>
 </resources>
 <connections/>
</ui>
